docs: add more content to "Get started" (#298)
bytebase/bytebase.com: content/docs/en/get-started/configure-the-workspace/overview.md

---
title: Configure the Workspace
---

You've already deployed Bytebase to proudction, before making any changes to databases, you should first configure the workspace.

> The entire Bytebase instance runs a single **Workspace**. At workspace level you can do fundamental settings applicable to all later created projects. There are three roles at this level: **Owner**, **DBA** and **Developer**. 

Reviewer: ```suggestion
The entire Bytebase instance hosts a single **Workspace**. At the workspace level, you can configure global settings applicable to all later created projects. There are three roles at the workspace level: **Owner**, **DBA**, and **Developer**. 
```
